### ISSUE TITLE
Is kaiwa alive?

### ISSUE THREAD

**Issue by Senset** [digicoop/kaiwa#96]
Hello! I'm trying to install kaiwa on Ubuntu 14.04 Server and have some problems with it. On step "npm install".
So before paste logs \ debugs I want to know: is Kaiwa alive (still have a people who can help?)

**Comment by jbreitbart** [comment]
Are there any good alternatives?

**Comment by Senset** [comment]
I'm using Jappix now (https://github.com/jappix/jappix). Installed on Ubuntu 14.04 Server, apache2+php v.5.5.9+prosody 0.9.10 via BOSH.
My server not in production, still testing... Seems to be OK

**Comment by djmaze** [comment]
> Notice: the Jappix project is no longer maintained. We still accept PR (Pull Requests) though.

**Comment by Senset** [comment]
Yep! But it works from box... that can not be said about the kaiwa, bcs my skills very low for nodejs (and npm) yet. And kaiwa creates more questions for me, like: "node server" command - how to create a service for it (cron via root... etc)? why it needs ldap lib (some manuals tells it: http://freedif.org/kaiwa-modern-xmpp-web-client/)? And more for talk: (imho) many users haves problem with install it (see the comments on link above)

**Comment by djmaze** [comment]
Yes, Kaiwa is also hardly maintained.  

I believe [Movim](https://movim.eu/) is the successor to Jappix. Probably worth trying it out.

**Comment by Senset** [comment]
Thx, I'll try

**Comment by dnut** [comment]
The problem with Movim is it seems to be attempting to be a social network, not just an XMPP client. It constantly connects to dozens of other servers and shares your information with them, which seems like a big privacy weakness when all you want is a client for your own users. If there were an easy way to strip it down to only being an XMPP client, I would definitely try it.

**Comment by rzr** [comment]
what about xmpp bridge for matrix.org ?

**Comment by jarbro** [comment]
Looks like this fork of Kaiwa is still getting development https://github.com/ForNeVeR/Kaiwa

**Comment by jcrben** [comment]
The README was updated to point out that it is unmaintained - looks like ForNeVeR's work is mostly happening on the develop branch https://github.com/ForNeVeR/Kaiwa/tree/develop

**Comment by alexanderadam** [comment]
@digicoop / @maximebf / @sebu77 maybe it would make sense to create an organisation (i.e. **kaiwacommunity** or **kaiwachat** or so).

You could add developers like @ForNeVeR, @albjeremias or @patelvanita360 to it and move this project [including it's website](https://github.com/digicoop/kaiwa-site) to that organization.

This way the website wouldn't point to the unmaintained project. :wink:

Because as an end user it is pretty difficult to chose from various and changing forks.

Or otherwise it might be better just to directly switch to [converse.js](https://github.com/conversejs/converse.js) (and maybe apply a fancier theme to it) because [it supports even encryption with OMEMO](https://github.com/conversejs/converse.js#features).

**Comment by rzr** [comment]
If there is any traction to have this project community maintained check:
https://github.com/abandonware/abandonware.github.io/issues/5

**Comment by Nyco** [comment]
Hey @all, good idea @alexanderadam and @rzr, maybe we can send a call for community maintenance? Probably, sometihng will happen?... :)

**Comment by ForNeVeR** [comment]
Probably it would be better to develop a new client based on the same underlying stanza.io, but built with more modern tech. Old good Kaiwa, while truly old and good, is hardly maintainable I'd say.

**Comment by alexanderadam** [comment]
> Kaiwa, while truly old and good, is hardly maintainable I'd say.

Okay, good to know.

> Probably it would be better to develop a new client

What's wrong with existing clients like [converse.js](https://github.com/conversejs/converse.js)?
It's actively maintained and has [modern features](https://github.com/conversejs/converse.js#features).
The default design is not my taste although [you can theme it](https://m.conversejs.org/docs/html/theming.html) by [using the mockup](https://conversejs.org/mockup/chatroom.html).

**Comment by maximebf** [comment]
Indeed we can create an organization. But as mentioned, I believe it has become hard to maintain and a major update of the dependencies would be needed.

**Comment by ForNeVeR** [comment]
converse.js looks very cool. If it supports modern tech, then I'd suggest that all possible Kaiwa contributors better focus their attention on it :)

**Comment by alexanderadam** [comment]
@maximebf / @sebu77 [could you please pin this issue](https://help.github.com/en/articles/pinning-an-issue-to-your-repository)?

Otherwise people somehow still seem to ignore that this project is unmaintained (although it should IMHO be clear by the [README](https://github.com/digicoop/kaiwa#kaiwa)).

Maybe it would also be better to change the title of this issue to something like `kaiwa is UNMAINTAINED` or so.

**Comment by Senset** [comment]
@alexanderadam, lol
I start that issue while unmaintained status wasn't declared in readme.
And now we know: it's dead =)

**Comment by alexanderadam** [comment]
@Senset I guess you misread my issue. :wink: 
I just wrote the comment because issues [like 107](https://github.com/digicoop/kaiwa/issues/107) were referenced last month and people who might stumble about it might won't see that this project is unmaintained.

**Your** issue on the other hand is probably the (only) one that's still relevant. :wink: 
So I would actually keep it open so that others can see that the project is unmaintained.

**Comment by Senset** [comment]
Okay

**Comment by Neustradamus** [comment]
Thanks for some messages about kaiwa, it will be nice to move to another organization...

**Comment by Neustradamus** [comment]
@Senset, how I can contact you?

**Comment by rzr** [comment]
Are any current maintainer able to join this org:
https://github.com/abandonware 
and then transfer the project (to keep history) ?

Any volunteer to help reviewing/merging pending patches ?

**Comment by maximebf** [comment]
The project has been moved to a new dedicated org. @Neustradamus is an owner of this org.
